ckanext/external_storage/templates/scheming/form_snippets/upload.html

```diff
@@ -0,0 +1,10 @@
+{% asset 'external_storage/FileInputComponentStyles' %}
+
+<div
+    id="FileInputComponent"
+    data-lfsServer="{{ h.extstorage_server_url() }}"
+    data-orgId="{{ h.extstorage_resource_storage_prefix(pkg_name).split('/')[0] }}"
+    data-datasetId="{{ pkg_name }}"
+></div>
+
+<script src="/external_storage/FileInputComponent.js?version=1"></script>
```

Comment:
Notice how i've added `?version=1` to the js file
It'll stop browsers using older cached versions of our js code